src/containers/InboxPage/InboxPage.js

```diff
@@ -168,6 +169,11 @@ export const InboxItem = props => {
             <FormattedMessage id="InboxPage.quantity" values={{ quantity }} />
           ) : null}
         </div>
+        {availabilityType == AVAILABILITY_MULTIPLE_SEATS ? (
+          <div className={css.itemSeats}>
+            <FormattedMessage id="InboxPage.seats" values={{ seats: unitLineItem.seats }} />
```

Comment:
This causes an error with transactions in inquiry state. (no line-items set at that point)
![Screenshot 2024-11-26 at 16 12 08](https://github.com/user-attachments/assets/25d33be4-fad0-4ca3-bedf-4cef0b0a4c5f)